### PR TITLE
feat(tuples): support importing tuples

### PR DESCRIPTION
**Pull request by rhamzeh**
## Description

All Failed (1 tuple per chunk):
Screenshot 2023-07-11 at 3 06 36 PM

Some Successful, Some Failed (1 tuple per chunk):
Screenshot 2023-07-11 at 3 06 21 PM

All failed (default 20 tuples per chunk)
Screenshot 2023-07-11 at 3 05 20 PM

## References

- #9 

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -336,13 +336,13 @@ fga model **validate**
...
+| [Delete Relationship Tuples](#delete-relationship-tuples)                         | `delete`  | `--store-id`, `--model-id`           | `fga tuple delete --store-id=01H0H015178Y2V4CX10C2KGHF4`                                                |
+| [Read Relationship Tuples](#read-relationship-tuples)                             | `read`    | `--store-id`, `--model-id`           | `fga tuple read --store-id=01H0H015178Y2V4CX10C2KGHF4 --model-id=01GXSA8YR785C4FYS3C0RTG7B1`            |
+| [Read Relationship Tuple Changes (Watch)](#read-relationship-tuple-changes-watch) | `changes` | `--store-id`, `--model-id`           | `fga tuple changes --store-id=01H0H015178Y2V4CX10C2KGHF4 --model-id=01GXSA8YR785C4FYS3C0RTG7B1`         |
+| [Import Relationship Tuples](#import-relationship-tuplesl)                        | `import`  | `--store-id`, `--model-id`, `--file` | `fga tuple import --store-id=01H0H015178Y2V4CX10C2KGHF4 --model-id=01GXSA8YR785C4FYS3C0RTG7B1`          |
```
Comment (adriantam) [inline README.md:345]: example should use --file

File: cmd/tuple/import.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+		}
+
+		tuples := []client.ClientTupleKey{}
+
+		data, err := os.ReadFile(fileName)
+		if err != nil {
+			return fmt.Errorf("failed to read file %s due to %w", fileName, err)
+		}
+
+		err = yaml.Unmarshal(data, &tuples)
```
Comment (adriantam) [inline cmd/tuple/import.go:124]: I thought we are also accepting JSON?

**Review by adriantam — CHANGES_REQUESTED** [review]

File: cmd/tuple/import.go (hunk 2 above)
Comment (rhamzeh) [reply cmd/tuple/import.go:124]: This handles both

File: cmd/tuple/import.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+		clientConfig := cmdutils.GetClientConfig(cmd)
+
+		fgaClient, err := clientConfig.GetFgaClient()
+		if err != nil {
+			return fmt.Errorf("failed to initialize FGA Client due to %w", err)
+		}
+
+		fileName, _ := cmd.Flags().GetString("file")
+
+		maxTuplesPerWrite, _ := cmd.Flags().GetInt("max-tuples-per-write")
```
Comment (adriantam) [inline cmd/tuple/import.go:107]: if we are testing for err, it should be 
```suggestion
		maxTuplesPerWrite, err := cmd.Flags().GetInt("max-tuples-per-write")
```

File: cmd/tuple/import.go [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+		if err != nil {
+			return fmt.Errorf("failed to parse max tuples per write due to %w", err)
+		}
+
+		maxParallelRequests, _ := cmd.Flags().GetInt("max-parallel-requests")
```
Comment (adriantam) [inline cmd/tuple/import.go:112]:
```suggestion
		maxParallelRequests, err := cmd.Flags().GetInt("max-parallel-requests")
```

File: cmd/tuple/import.go [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+		if err != nil {
+			return fmt.Errorf("failed to parse max tuples per write due to %w", err)
```
Comment (adriantam) [inline cmd/tuple/import.go:114]:
```suggestion
			return fmt.Errorf("failed to parse max parallel requests due to %w", err)
```

File: cmd/tuple/import.go [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+		if err != nil {
+			return err
+		}
+
+		return output.Display(*result) //nolint:wrapcheck
+	},
+}
+
+func init() {
+	importCmd.Flags().String("file", "", "Tuples File")
```
Comment (adriantam) [inline cmd/tuple/import.go:139]:
```suggestion
	importCmd.Flags().String("file", "", "Tuples file")
```

File: README.md [hunk 7]
```diff
@@ -443,6 +443,69 @@ fga tuple **changes** --type <type> --store-id=<store-id>
...
+
+###### Parameters
+* `--store-id`: Specifies the store id
+* `--model-id`: Specifies the model id to target (optional)
+* `--file`: Specifies the file name, `yaml` and `json` files are supported
+* `--max-tuples-per-write`: Max tuples to send in a single write (optional, default=20)
+* `--max-parallel-requests`: Max requests to send in parallel (optional, default=4)
+
+File format should be:
+In Yaml:
```
Comment (adriantam) [inline README.md:459]:
```suggestion
In YAML:
```

File: README.md (hunk 1 above)
Comment (rhamzeh) [reply README.md]: Done

**Review by adriantam — APPROVED** [review]

**Pull request merged**